sys/test_utils/dummy_thread/dummy_thread.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2020 Kaspar Schleiser <kaspar@schleiser.de>
+ *                    Freie Universität Berlin
+ *                    Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_test_utils
+ * @{
+ *
+ * @file
+ * @brief       Module creating a dummy thread
+ *
```

Comment:
As always, some documentation here about why we need this module and how to use it would be good to have